### ISSUE TITLE
Going between communities doesn’t keep the previously active/selected address

### ISSUE THREAD

**Issue by sachben91** [hicommonwealth/commonwealth#5050]
### Environment 
- Chrome on desktop 

### Steps to replicate
- Join a community, check that the address you want is connected to the community
- navigate to another community and navigate back to the previous one 

[Loom](https://www.loom.com/share/a48fa9cb9d1747e88cf02c127920f9c1?sid=795637b8-6ec0-4720-a571-f56505387b79)

### Actual Result
- User is prompted to join the community again even though they joined the community earlier 

### Expected Result
- The address that the user selected the last time they were on the community should be the default unless the session has expired 

### Priority for launch
-  blocking the launch since this causes a sub-par user experience that the user has no way around

**Comment by masvelio** [comment]
Reproducible. Persistence has been introduced [lately here](https://github.com/hicommonwealth/commonwealth/pull/4907) and it worked correctly. It seems that for some reason it stopped. Persistency does not work neither on production. It needs deeper investigation.

**Comment by kurtisassad** [comment]
Previous deploy might have fixed this. @sachben91 Could you retry this? I had tried it on frack but cannot get the join community to show up after already having join the community. Steps I used to recreate:

Join dydx
Join theresearchcow
Add 2nd address
Favourite dydx and theresearchcow to show up in sidebar
Navigate to dydx/discussions
Switch Account to 2nd account
click on theresearchcow through sidebar
click on dydx through sidebar

Join button is greyed out for me.

**Comment by masvelio** [comment]
issue fixed with this commit https://github.com/hicommonwealth/commonwealth/pull/3426/commits/7cc344a0cc1ffb9dd24db77f1563b23c7ee7b290

**Comment by CowMuon** [comment]
I don't think this was fixed in a previous commit as of when Kurtis made above comment. Rather, this appears to be fixed in commit 7cc344a0c which went in the next day.
